### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by zairdon20**
In the following milestone i made sure i implemented the following:

- The home page should have a link in the menu to the about page.

- The logo in the header links to the home page.

- Mobile menu

- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.

- The mobile menu should have a close (X) button that closes the menu.

- Dynamic page

- The section "Featured speakers" has created dynamically in JavaScript.

- I used a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,64 +1,49 @@
...
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
+## Demo Video Link
 
+[Watch Demo](https://zairdon20.github.io/Last-capstone/)
```
Comment (rustam-tolipov) [inline README.md]: - [ ] Nice and descriptive readme👍. However, Your links are opening wrong pages, for example,[ live demo]() a link is opening 404 error Github page, and the [ Watch demo](https://zairdon20.github.io/Last-capstone/) a link is opening the live demo of your project 😊🍀

File: index.html [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+              to be inefficient
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="allPrograms">
+        <h3>JOIN THE EVENT</h3>
+        <hr />
+      </div>
+    </section>
```
Comment (rustam-tolipov) [inline index.html]: - [ ] AWESOME JOB 💎 so far but there are some things that need to be fixed and I am sure you can fix it in no time ⌚. In this section titles are creating a new line please kindly fix this issue 💯👍

Screen Shot 2021-12-25 at 5 24 33 PM

File: index.html [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+
+        <div class="partnersMain">
+          <div class="partnersItem">
+            <img
+              src="../capston-project/imgs/intel.png"
+              width="250"
+              height="70"
+              alt="Open AI"
+              class="partner_img"
+            />
```
Comment (rustam-tolipov) [inline index.html]: - [ ] One of your partners is missing 🕵🏻‍♂️, I mean one image is not working but with that, We tested that alt is working well. Also, fix this issue and your project will be completed 🍀

Screen Shot 2021-12-25 at 5 22 50 PM

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @zairdon20 , 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (zairdon20) [reply README.md]: thank you

File: index.html (hunk 3 above)
Comment (zairdon20) [reply index.html]: thank you

File: index.html [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+          </p>
+        </div>
+        <div class="programCards">
+          <i class="fa fa-rocket fa-2x"></i>
+          <div><h3>Exhibition</h3></div>
+          <p>
+            Many national governments around the world are showing themselves to
+            be inefficient
+          </p>
+        </div>
```
Comment (MrBrN197) [inline index.html:90]: - [ ] Thank you for adjusting the CSS for the main program section. please notice that now the text is breaking outside the container. Please adjust the width of each container to ensure that this doesn't happen. 
## Your Design notice the breaking. 
image

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @zairdon20  👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
```
Comment (MrBrN197) [inline index.html:90]: - [ ] Please look at my previous comment. the text is still breaking out of the container. I see you've set the text to a smaller size. please make sure that this CSS is the one that is active on the desktop version.

**Review by MrBrN197 — CHANGES_REQUESTED** [review]
## Hi, @zairdon20 👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MrBrN197 — APPROVED** [review]
## Hi,  @zairdon20  👋

Hi, thank you for making the changes I request. it seems there was a problem seeing the latest changes. but I can see that you've made the correct changes.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**